Description: Don't install compatibility symlinks
Forwarded: not-needed
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2012-02-19
Index: spectrwm.git/linux/Makefile
===================================================================
--- spectrwm.git.orig/linux/Makefile	2012-09-16 16:52:29.283080884 +0200
+++ spectrwm.git/linux/Makefile	2012-09-16 16:53:24.923999892 +0200
@@ -47,7 +47,6 @@
 	install -m 755 -d $(DESTDIR)$(MANDIR)/ru/man1
 	install -m 755 -d $(DESTDIR)$(XSESSIONSDIR)
 	install -m 755 spectrwm                    $(DESTDIR)$(BINDIR)
-	ln -sf spectrwm                            $(DESTDIR)$(BINDIR)/scrotwm
 	install -m 644 libswmhack.so.$(LIBVERSION) $(DESTDIR)$(LIBDIR)
 	install -m 644 ../spectrwm.1               $(DESTDIR)$(MANDIR)/man1/spectrwm.1
 	install -m 644 ../spectrwm_es.1            $(DESTDIR)$(MANDIR)/es/man1/spectrwm.1
@@ -58,7 +57,6 @@
 
 uninstall:
 	rm -f $(DESTDIR)$(BINDIR)/spectrwm
-	rm -f $(DESTDIR)$(BINDIR)/scrotwm
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBVERSION)
 	rm -f $(DESTDIR)$(MANDIR)/man1/spectrwm.1
 	rm -f $(DESTDIR)$(MANDIR)/es/man1/spectrwm.1
